### ISSUE TITLE
Frames, node groups, "value" node

### ISSUE THREAD

**Issue by ZackMercury** [nomelif/Audionodes#18]
I know, this is not supposed to be a serious project, but it got me super excited with what you can do with it, it's just super fascinating and I'd like to make a few tutorials, but I can't make my nodes look better, sort and group them etc, basic node editor functionality is missing, so it may be difficult to understand what's going on for people trying to figure out your setup, since you can't group nodes into node groups with certain inputs.
I also find it a little annoying that you can't do such basic stuff

**Comment by ZackMercury** [comment]
Also would be nice to have the "Node Wrangler" functionality here too.
I mean, Ctrl+Shift+Click on a node would connect it to the output

**Comment by ollpu** [comment]
I agree. We haven't had as much time to spend on the project as we'd hope, and have prioritized other features. Node groups especially have been in the plans for a long time.

**Comment by ollpu** [comment]
Oh, and by the way, frames do work (e.g. `Ctrl-J`), they just aren't available in the Add Node menu.
